### ISSUE TITLE
Some Information before promting camera access

### ISSUE THREAD

**Issue by e1mo** [krmax44/nutri.cam#1]
I is kinda creepy when a website prompts you for access to your webcam without any context. It would be cool to have at least some bullet points why the camera access is required and a button like "Grant access" which then would trigger the webcam.

**Comment by krmax44** [comment]
Definitely. It also might be nice to have a mode that works even with no camera.
